packages/next-swc/crates/next-core/src/next_client/context.rs

```diff
@@ -53,17 +53,25 @@ use crate::{
     util::foreign_code_context_condition,
 };
 
-pub fn next_client_defines() -> CompileTimeDefinesVc {
+fn defines() -> CompileTimeDefines {
     compile_time_defines!(
         process.turbopack = true,
         process.env.NODE_ENV = "development",
-        process.env.__NEXT_CLIENT_ROUTER_FILTER_ENABLED = false
+        process.env.__NEXT_CLIENT_ROUTER_FILTER_ENABLED = false,
+        process.env.__NEXT_HAS_REWRITES = true,
+        process.env.__NEXT_I18N_SUPPORT = false,
```

Comment:
This is still missing quite a lot, can we re-use https://github.com/vercel/next.js/blob/ebddbf1b00bdd28bd9728278065645e1714dc890/packages/next/src/build/webpack-config.ts#L179 to ensure we're matching?

Reply:
Yeah, we can do that, but that requires some refactoring... I created an issue for it and added a TODO